tests/utils/testutils.py

```diff
@@ -219,13 +219,13 @@ def setUpClass(cls):
         else:
             cls.host_stdout = tempfile.NamedTemporaryFile('w+t')
 
-        _setup_func_app(TESTS_ROOT / script_dir)
         try:
             if is_envvar_true(CONSUMPTION_DOCKER_TEST):
```

Comment:
How would this work? we are trying to find a boolean, but now the name has been changed in constants.py?

Reply:
We provide a boolean in the github workflow yml. The is_env_true willl check if the value of the env variable provided in the github yml file is true or not. If true it will run the corresponding docker tests and if not it will run the legacy tests as usual